packages/core/src/renderer/components/entity-settings/extension-registrator.injectable.ts

```diff
@@ -36,6 +39,7 @@ export interface RegisteredEntitySetting {
   components: EntitySettingComponents;
   source?: string;
   group: string;
+  isShown: IComputedValue<boolean>;
```

Comment:
Why not call this also `visible`?

Reply:
I tried to replicate the existing behavior that was implemented with `AppPreferenceTabRegistration`. I think it's a convention in this code base for `Showable` and such